pathz/pathz.proto

```diff
@@ -205,27 +155,89 @@ message UploadRequest {
   // memory for preservation across system reboots.
   // `created_on` is a timestamp: the number of seconds since
   // January 1st, 1970 00:00:00 GMT a.k.a. unix epoch.
-  uint64 created_on = 3;
+  uint64 created_on = 2;
 
-  // The actual OpenConfig Path-based Authz Policy.
-  gnsi.authorization.AuthorizationPolicy policy = 4;
+  // The actual OpenConfig gNMI Path-based Authorization Policy.
+  gnsi.authorization.AuthorizationPolicy policy = 3;
 }
 
 message UploadResponse {
 }
 
-// ProbeRequest contains a single user name and gNMI Path being attempted.
-// Data returned to an RPC Caller should adhere to the policy.
+enum PolicyInstance {
+  // Invalid instance. Referring to this instance in any of the RPCs always
+  // results in an error report.
+  POLICY_INSTANCE_UNSPECIFIED = 0;
+
+  // The policy that is currently used by the gNMI service to authorize access.
+  POLICY_INSTANCE_ACTIVE = 1;
+
+  // The most recent policy that has been uploaded during the Rotation() RPC.
+  // If there is no Rotate() RPC in progress, then referring to this instance
```

Comment:
Is the concept of "on Rotate() RPC in progress" meant to be per server?  ie, to support one client performs a rotation and another client tries to probe at the same time.  Otherwise, use of Sandbox can be implicitly assumed if the client had issued any rotate (which I believe is the theme in the other gNSI RPC that support test before commit).

Reply:
I take what you said as each stream has its own independent sandbox then.  I wonder if that is worth being specific in the comment?  ...just in case someone tries to write a client that use one stream for rotation and a separate one for probing.